noxfile.py

```diff
@@ -77,6 +77,8 @@ def run_integration(session):
 @nox.session(name="doctests")
 def run_doctests(session):
     """Run the doctests and generate the output(s) in the docs/build/ directory."""
+    # TODO: Temporary fix for Python 3.12 CI.
```

Comment:
```suggestion
    # TODO: Temporary fix for Python 3.12 CI.
    # See: https://bitbucket.org/pybtex-devs/pybtex/issues/169/
```

Let's add a comment that links to the original issue. The issue is fixed with this hack from their end, but we are yet to see a release with that fix